server/src/main/java/org/opentosca/toscana/plugins/cloudformation/util/CloudFormationScript.java

```diff
@@ -10,11 +10,16 @@
 
 import static org.opentosca.toscana.plugins.cloudformation.CloudFormationModule.FILEPATH_TARGET;
 
+/**
+ Class that overrides the {@link BashScript} implementation.
```

Comment:
This comment does not provide anything, it just repeats the class header. A javadoc  comment should not explain `what` or `how` (if you're code fails to provide this information, refactor your code), it should always focus on the `why`.